centrosome/neighmovetrack.py

```diff
@@ -85,15 +91,15 @@ def from_labels(labels):
         """
         labels = labels.astype(int)
 
-        areas = scipy.ndimage.measurements.sum(labels != 0, labels, range(1, numpy.max(labels) + 1))
+        areas = scipy.ndimage.measurements.sum(labels != 0, labels, list(range(1, numpy.max(labels) + 1)))
 
         existing_labels = [i for (i, a) in enumerate(areas, 1) if a > 0]
 
         existing_areas = [a for a in areas if a > 0]
 
         existing_centers = scipy.ndimage.measurements.center_of_mass(labels != 0, labels, existing_labels)
 
-        zipped = (zip(existing_labels, existing_centers, existing_areas))
+        zipped = (list(zip(existing_labels, existing_centers, existing_areas)))
```

Comment:
Both the surrounding parenthesis and the `list` cast can be removed from here and above